include/albatross/src/models/sparse_gp.hpp

```diff
@@ -136,27 +136,27 @@ class SparseGaussianProcessRegression
   };
 
   void initialize_params() {
-    log_measurement_nugget_ = {log10(details::DEFAULT_NUGGET),
-                               std::make_shared<UninformativePrior>()};
-    log_inducing_nugget_ = {log10(details::DEFAULT_NUGGET),
-                            std::make_shared<UninformativePrior>()};
+    measurement_nugget_ = {details::DEFAULT_NUGGET,
```

Comment:
Should this be the new type,`LogScaleUniformPrior` ?

Reply:
Yeah, so I did that as first ... but then found it hard to assign an upper bound that would apply to all applications.  I suppose I could do something like `[1e-16, 1e16]` and that should be generally applicable (and if not users can always overrwrite the default prior.

Reply:
I've followed up, switching to `LogScaleUniformPrior` with the numerical limits of the double as bounds (ie, as close to zero as possible and as large as possible).